dbms/src/TiDB/Schema/tests/gtest_schema_sync.cpp

```diff
@@ -179,6 +178,14 @@ class SchemaSyncTest : public ::testing::Test
     Context & global_ctx;
 };
 
+TEST_F(SchemaSyncTest, SchemaDiff)
+try
+{
+    SchemaDiff diff;
+    std::string data = "{\"version\":40,\"type\":31,\"schema_id\":69,\"table_id\":71,\"old_table_id\":0,\"old_schema_id\":0,\"affected_options\":null}";
```

Comment:
```suggestion
    // Note that we should keep backward compatibility for parsing old version schema diff.
    // Or it will lead to exception thrown after upgraded.
    std::string data = "{\"version\":40,\"type\":31,\"schema_id\":69,\"table_id\":71,\"old_table_id\":0,\"old_schema_id\":0,\"affected_options\":null}";
```